frontend/src/features/public-form/utils/createSubmission.ts

```diff
@@ -78,20 +78,30 @@ export const createEncryptedSubmissionData = async ({
 }
 
 /**
+ * Used for both Email mode submissions and Storage mode submissions after encryption boundary shift.
  * @returns formData containing form responses and attachments.
  * @throws Error if form inputs are invalid.
  */
-export const createEmailSubmissionFormData = (
+export const createClearSubmissionFormData = (
   formFields: FormFieldDto[],
   formInputs: FormFieldValues,
   responseMetadata?: ResponseMetadata,
+  setBody?: {
```

Comment:
suggestioN: maybe something like `additionalBodyFields` might be clearer than `setBody`

Reply:
Thanks for the better nomenclature suggestion! Done here: [de584a9](https://github.com/opengovsg/FormSG/pull/6665/commits/de584a90f555b5e56344c1440a272a258cc8b562)